charts/ceph-csi-cephfs/values.yaml

```diff
@@ -38,7 +38,11 @@ nodeplugin:
   name: nodeplugin
   # if you are using ceph-fuse client set this value to OnDelete
   updateStrategy: RollingUpdate
-
+  # if you are using ceph-fuse client,
+  # delete the below mentioned rolling update strategy.
```

Comment:
why delete? would it not help to change the values? if you delete this, should `updateStrategy` not be changed too?